src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/RazorLSPOptionsMonitorTest.cs

```diff
@@ -24,7 +24,7 @@ public RazorLSPOptionsMonitorTest()
         public async Task UpdateAsync_Invokes_OnChangeRegistration()
         {
             // Arrange
-            var expectedOptions = new RazorLSPOptions(Trace.Messages, enableFormatting: false, autoClosingTags: true);
+            var expectedOptions = new RazorLSPOptions(Trace.Messages, enableFormatting: false, autoClosingTags: true, insertSpaces: true, tabSize: 4);
```

Comment:
I expected some settings other than `insertSpaces:true, tabSize: 4` to be tested here, is the actual value of the properties just not doing anything interesting at this layer?